src/styles/mo-components/_fixes.scss

```diff
@@ -200,3 +200,11 @@ body.donate-page {
     }
   }
 }
+
+/* Homepage join form fixes */
+.subscribe-sms-conditions {
```

Comment:
Since we are controlling the html on the homepage, wouldn't it be better just to have a `display:none` inline on that side?  If we have it in here, doesn't that make it difficult to display when/if we did want to?

Reply:
It still works as long as the `unhide` class appears after this one in the CSS. I could move the `display: none` to inline styling.